### PR TITLE
Fix path for shared translation files

### PR DESCRIPTION
**Pull request by lennym**
This needs to have the `src/` on the end or it will throw when it finds .js files in the parent directory.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "cover": "istanbul cover _mocha && istanbul check-coverage",
     "sass": "./node_modules/node-sass/bin/node-sass ./assets/scss/app.scss ./public/css/app.css --include-path ./node_modules",
     "create:public": "mkdir -p ./public/js ./public/css ./public/images",
-    "hof-transpile": "hof-transpiler ./apps/**/translations/src -w --shared ./node_modules/hof-template-partials/translations",
+    "hof-transpile": "hof-transpiler ./apps/**/translations/src --shared ./node_modules/hof-template-partials/translations/src",
```
Comment (sulthan-ahmed) [inline package.json]: just curious it doesn't need the -w anymore?

Comment (sulthan-ahmed) [inline package.json]: I looked at the documentation looks like it doesn't need it anymore https://github.com/UKHomeOfficeForms/hof-transpiler/commit/e4a8bfc2138d760628bb14b5cb6a1e0298941991#diff-04c6e90faac2675aa89e2176d2eec7d8L17

Comment (lennym) [inline package.json]: I don't *think* it ever did.

Comment (sulthan-ahmed) [inline package.json]: Curious how it came to have it.  Thanks for clearing that up

Comment (lennym) [inline package.json]: Indiscriminate copy-pasting without regard for content probably. Most things are.

**Pull request merged**